client/app/components/elements/Avatar/Avatar.css

```diff
@@ -6,4 +6,14 @@
 .link {
   display: inline-block;
   height: 100%;
+  width: 100%;
+}
+
+.textAvatar {
+  width: 100%;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
```

Comment:
Color could come from variables.js
Have you thought about font-sizes? Should the font size be what ever is defined outside of Avatar or should it be proportional to avatar components width (or height). That way it would scale always depending on how big avatar we are going to use.

(The latter would mean either direct % value, or [css lock formula](http://blog.typekit.com/2016/08/17/flexible-typography-with-css-locks/) The idea behind the formula is that there's defined min & max, but the value can increase within those bounds.)

I'm not sure which one is better. In Topbar, it makes sense that the font-size is the same as in other elements. However, in ListingCard it might make more sense that the size is proportional (and in ListingPage design there was some 3rd size of Avatar used).